raiden/transfer/mediated_transfer/events.py

```diff
@@ -231,7 +232,7 @@ def from_dict(cls, data: Dict[str, Any]) -> 'SendSecretReveal':
             recipient=to_canonical_address(data['recipient']),
             channel_identifier=ChannelID(int(data['channel_identifier'])),
             message_identifier=MessageID(int(data['message_identifier'])),
-            secret=Secret(serialization.deserialize_bytes(data['secret'])),
+            secret=Secret(deserialize_bytes(data['secret'])),
```

Comment:
why not have a `deserialize_secret`  function as for all other types?

Reply:
I do have one. Shjould have been used here. Good catch